.pylintdict

```diff
@@ -1,41 +1,57 @@
 acc
+adaptively
 admmstate
+aer
 aer's
 al
 annealers
+applegate
 args
+arxiv
 autosummary
 backend
-bb
+behaviour
```

Comment:
This one in the Beitish English spelling - since we are using US English it would be preferable to be removed and fixed at source to be `behavior`

Reply:
It is fixed